easybuild/easyconfigs/e/ExaBayes/ExaBayes-1.5-foss-2016b.eb

```diff
@@ -0,0 +1,31 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = 'ExaBayes'
+version = '1.5'
+
+homepage = 'https://cme.h-its.org/exelixis/web/software/exabayes/'
+description = """ExaBayes is a software package for Bayesian tree inference.
+ It is particularly suitable for large-scale analyses on computer clusters"""
+
+# only tested with GCC < 6
+# https://groups.google.com/d/msg/exabayes/94tOvKRDq-s/F223g6qKCQAJ
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://cme.h-its.org/exelixis/resource/download/software/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e401f1b4645e67e8879d296807131d0ab79bba81a1cd5afea14d7c3838b095a2']
+
+configopts = '--enable-mpi '
```

Comment:
If you have enable-mpi, shouldn't you use toolchainopts 'usempi': True too?

Reply:
As a matter of fact, yes you do need 'usempi': True, so it will find the correct MPICXX command.